third_party/move/move-vm/integration-tests/src/compiler.rs

```diff
@@ -55,8 +56,24 @@ fn compile_source_unit_v2(
     Ok(units)
 }
 
+pub fn compile_units_move_2_2(s: &str) -> Result<Vec<AnnotatedCompiledUnit>> {
```

Comment:
We could call this `compile_units_with_move_version()`, passing the language version explicitly. Otherwise, we may need to start creating one such function for each new language version.

Also, these changes would not be necessary if we do the move-compiler-v2 transactional tests instead.

Reply:
These changes are no longer needed, removed